waspc/cli/Wasp/Cli/Command/Common.hs

```diff
@@ -57,3 +61,28 @@ alphaWarningMessage :: String
 alphaWarningMessage =
   "NOTE: Wasp is still in Alpha, therefore not yet production ready "
     ++ "and might change significantly in the future versions."
+
+toWaspMessage :: String -> String
+toWaspMessage = waspMessage ""
+
+toWaspStartMessage :: String -> String
+toWaspStartMessage = waspMessage "🐝"
+
+toWaspSuccessMessage :: String -> String
+toWaspSuccessMessage = waspMessage "✅"
+
+toWaspFailureMessage :: String -> String
+-- Add a bit more padding on errors for more pronounced
+-- visibility and better display of any following error context.
+toWaspFailureMessage str = "\n" ++ waspMessage "❌" str ++ "\n"
+
+waspMessage :: String -> String -> String
+waspMessage emoji message = unwords ["\n", prefix, message, suffix, "\n"]
```

Comment:
`unwords` here introduces a whitespace betweeen the first `\n` and `prefix`, therefore resulting in all the messages in the terminal being indented for 1 whitespace -> is that smth we want to do on purpose or is that an accident?

Reply:
That was done on purpose so I didn't have to pad the `prefix ++ " " ++ message + " " + suffix` but it may be muddled with the "\n" in there. The byproduct is I also liked the space before the prefix visually too.

If we are ok with nixing the leading space, I think the question is what would be the best way to get:
```haskell
"\n" ++ prefix ++ " " ++ message ++ " " ++ suffix ++ "\n"
```
I can tack it on to the end of prefix and start of suffix, but was trying to show the spacing intent with `unwords` but can go either way.

Reply:
Yeah this works and is probably easier to understand the intent: `concat ["\n", prefix, " ", message, " ", suffix, "\n"]` Thanks 👍🏻

Reply:
Updated here: https://github.com/wasp-lang/wasp/commit/526f6e93b45a88ddf99d59ac3bb92d1f244a8d59